### PR TITLE
Gateway page improvements

### PR DESCRIPTION
**Pull request by fadeev**
* Added links to other docs
* Added diagram

## Summary by CodeRabbit

- **New Features**
	- Introduced an embedded Figma design for the Gateway interface.
	- Added a new section on "Revert Handling" for cross-chain operations.
  
- **Documentation**
	- Updated text with hyperlinks for more information on EVM, Solana, and Bitcoin functionalities.
	- Minor textual adjustments for clarity and additional resources.

### CODE REVIEW

**Comment by fadeev** [comment]
@zeta-chain/fullstack please, review.

File: src/pages/developers/evm/gateway.mdx [hunk 1]
```diff
@@ -1,6 +1,14 @@
 Gateway is an interface that serves as a unified entry point for interactions
 between contracts on connected chains and universal apps on ZetaChain.
 
+<iframe
```
Comment (hernan-clich) [inline src/pages/developers/evm/gateway.mdx:4]: The Figma diagram looks really cool, I'm wondering if they have any query params or options to remove the footer of the diagram that redirects to the actual Figma page, for one I don't think "Untitled" is a very fitting name.

image

But I also don't think it makes sense to display the names of the core contributors that are currently viewing the file.

image

Comment (fadeev) [reply src/pages/developers/evm/gateway.mdx:4]: Thanks for feedback! Removed footer and renamed the file [hide footer](https://github.com/zeta-chain/docs/pull/518/commits/0b8342d5e7505f3da4abe339ae3b40c357de4053). Removing viewers would be nice, but it's a Figma account config, so we can't really fix it in docs source. I'll ask the team if it's something we can change.

**Review by hernan-clich — APPROVED** [review]
Tested ACK

**Review by GMaiolo — APPROVED** [review]

**Pull request merged**